test/state/precompiles.cpp

```diff
@@ -292,6 +293,35 @@ ExecutionResult blake2bf_execute(const uint8_t* input, [[maybe_unused]] size_t i
     return {EVMC_SUCCESS, sizeof(h)};
 }
 
+namespace
+{
+using intx::operator""_u256;
+constexpr auto FIELD_ELEMENTS_PER_BLOB = 4096_u256;
+constexpr auto BLS_MODULUS =
+    52435875175126190479447740508185965837690552500527637822603658699938581184513_u256;
+}  // namespace
+
+ExecutionResult point_evaluation_execute(const uint8_t* input, size_t input_size, uint8_t* output,
+    [[maybe_unused]] size_t output_size) noexcept
+{
+    assert(output_size >= 64);
```

Comment:
Is it not == instead of >= ?

Reply:
Oh the output could have came from something that allocated more than 64 bytes, the caller can just take the first 64 bytes

Reply:
These are internals of the precompiles framework. It is expected the caller to allocate at least 64 for the output.